src/main/java/com/jalasoft/selenium/dmoreno/movies/Customer.java

```diff
@@ -0,0 +1,61 @@
+package com.jalasoft.selenium.dmoreno.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ *
+ */
+public class Customer {
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * @param name name
+     */
+    public Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * @param rental rental
+     */
+    public void addRental(final Rental rental) {
+        rentals.addElement(rental);
+    }
+
+    /**
+     * @return name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * @return statement
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things. it is calculating the totalAmount, totalFrequentRenterPoints and printing.